src/cli.rs

```diff
@@ -16,7 +16,7 @@ impl Cli {
         let mut config = E4EDMConfig::build().unwrap();
         let _ = match &self.command {
             Commands::InitDataset(args) => init_dataset(args, &mut config),
-            Commands::InitMission(args) => bail!("unimplemented"),
+            Commands::InitMission(args) => init_mission(args, &mut config),
```

Comment:
From my testing the errors from the command functions currently aren't being caught. This can be fixed by adding a `?` operator on line 26.

Reply:
For the record, I tested this just by attempting to make a new mission with all the requisite args - since we don't have the ability to load configs at the moment the `init_mission` command should always fail.

Reply:
Fixed this -- also adjusted line 28 `let _ = config.save();` to `config.save()?` as a reminder that an error there needs to be handled in the future.